eng/targets/Imports.targets

```diff
@@ -154,6 +154,14 @@
                  Condition="$(_VersionComparisonResult) &lt; 0"/>
   </Target> 
 
+  <Target Name="_CheckLongPathSupport" BeforeTargets="BeforeBuild" Condition="'$(MSBuildRuntimeType)' == 'Full'">
+    <PropertyGroup>
+      <_RoslynLongPathsEnabled>$([MSBuild]::GetRegistryValueFromView('HKEY_LOCAL_MACHINE\System\CurrentControlSet\Control\FileSystem', 'LongPathsEnabled', null, RegistryView.Registry64, RegistryView.Registry32))</_RoslynLongPathsEnabled>
+    </PropertyGroup>
+    
+    <Warning Condition="'$(_RoslynLongPathsEnabled)' != '1'" Text="Long paths are required for this project. Please run eng\enable-long-paths.reg" />
```

Comment:
💡 Consider assigning `Code` here as well

Reply:
Can you elaborate on that? Not quite sure what that means.

Reply:
something like
```
<Warning Code="JARED001" Condition="'$(_RoslynLongPathsEnabled)' != '1'" Text="Long paths are required for this project. Please run eng\enable-long-paths.reg" />
```
personally I don't see particular value unless you're going to write help content for it or expect people to suppress it

Reply:
Yeah I specifically don't want this warning to be suppressible.